sklearn/ensemble/tests/test_stacking.py

```diff
@@ -31,7 +31,12 @@
 from sklearn.tree import DecisionTreeRegressor
 from sklearn.ensemble import RandomForestClassifier
 from sklearn.ensemble import RandomForestRegressor
+from sklearn.experimental import enable_hist_gradient_boosting  # noqa
+from sklearn.ensemble import HistGradientBoostingClassifier,\
+    HistGradientBoostingRegressor
```

Comment:
```suggestion
from sklearn.ensemble import HistGradientBoostingClassifier
from sklearn.ensemble import HistGradientBoostingRegressor
```

Reply:
I think this is an overkill to use HistGradientBoosting for a smoke test. We could simply use a LinearRegression and a LogisticClassifier (or something even faster)

Reply:
Agree, I will change the place where I used HistGradBoosting. 
Similar thought I had, that when I started using LR in few places.